### ISSUE TITLE
Add bpython package

### ISSUE THREAD

**Issue by AnderRasoVazquez** [termux/termux-packages#35]
[bpython](http://bpython-interpreter.org/) is a fancy interface to the Python interpreter for Linux, BSD, OS X and Windows (with some work). bpython is released under the MIT License. It has the following (special) features:

- In-line syntax highlighting
- Readline-like autocomplete with suggestions displayed as you type.
- Expected parameter list for any Python function.
- "Rewind" function to pop the last line of code from memory and re-evaluate.
- Send the code you've entered off to a pastebin.
- Save the code you've entered to a file.
- Auto-indentation.
- Python 2 and 3 support.

[SCREENSHOTS](http://bpython-interpreter.org/screenshots)
[SHORT VIDEO](http://bpython-interpreter.org/videos/01.ogg)

On my Arch Linux I have installed these versions:
- For Python2: bpython2 (I think it includes also bpython2-curses and bpython2-urwid)
- For Python3: bpython (I think it includes also bpython-curses and bpython-urwid)

Keep up the good work! Termux is the best terminal emulator I've tried for Android, installing packages with apt is a killer feature =D

**Comment by fornwall** [comment]
Thanks for the feedback!

Since bpython is available from the pip package manager I think that is the best way to install it: Install the dependencies with `apt install gcc python-dev`, then just run `pip install bpython`. Does that work for you? Re-open the issue if otherwise!

**Comment by AnderRasoVazquez** [comment]
So that was the problem! I didn't install those dependencies. Thank you @fornwall . I should add that for python2 bpython version would be:
`$ apt install gcc python2-dev`
`$ pip install bpython`

**Comment by AnderRasoVazquez** [comment]
Nope, bpython for python2 doesn't run. Reopen this issue please.

![photo_2015-11-13_20-41-39](https://cloud.githubusercontent.com/assets/3936719/11155666/339cfdb0-8a47-11e5-8ac1-95f961d8a3e3.jpg)


Also, I think I cannot install both versions. I mean bpython for python3 and python2 because they have the same name.

**Comment by fornwall** [comment]
Does running `export PYTHONIOENCODING=UTF-8` before starting bpython work in python2?

**Comment by AnderRasoVazquez** [comment]
Yes, now it works! But now I don't know how can I have those two versions installed. When I do `which bpython` both versions will point to `/data/data/com.termux/files/usr/bin/bpython`. How can I fix that?

**Comment by fornwall** [comment]
I've just updated the python2 package for improved unicode support, so setting PYTHONIOENCODING should no longer be necessary after doing a `apt update && apt upgrade`!

To have both versions installed I used the following:

    pip2 install bpython
    cp $PREFIX/bin/bpython $PREFIX/bin/bpython2
    pip3 install bpython

After running that one can use `bpython2` for the python2 version as that was stored away by the `cp` command, and use `bpython` for the python3 version!

I'm closing this issue, re-open if you have any more problems! Thanks for your help in improving the python packages!
